RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py

```diff
@@ -68,15 +72,190 @@
            excludeOtherSeeds = cms.bool(True),
            minFracTot = cms.double(1e-20), ## numerical stabilization
            recHitEnergyNorms = cms.VPSet(
-           cms.PSet( detector = cms.string("HCAL_BARREL1"),
-                     recHitEnergyNorm = cms.double(0.8)
-                     ),
-           cms.PSet( detector = cms.string("HCAL_ENDCAP"),
-                     recHitEnergyNorm = cms.double(0.8)
-                     )
-           )
+            cms.PSet( detector = cms.string("HCAL_BARREL1"),
+                      depths = cms.vint32(1, 2, 3, 4),
+                      recHitEnergyNorm = cms.vdouble(0.8, 0.8, 0.8, 0.8),
+                      ),
+            cms.PSet( detector = cms.string("HCAL_ENDCAP"),
+                      depths = cms.vint32(1, 2, 3, 4, 5, 6, 7),
+                      recHitEnergyNorm = cms.vdouble(0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8),
+                      )
+            )
     ),
     positionReCalc = cms.PSet(),
     energyCorrector = cms.PSet()
 )
 
+#####
+
+seedFinderThresholdsByDetector2017 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+
+seedFinderThresholdsByDetector2018 = cms.VPSet(
+    cms.PSet(
+        detector = cms.string("HCAL_BARREL1"),
+        depths = cms.vint32(1, 2, 3, 4),
+        seedingThreshold = cms.vdouble(1.0, 1.0, 1.0, 1.0),
+        seedingThresholdPt = cms.vdouble(0.0, 0.0, 0.0, 0.0)
+        ),
+    cms.PSet(
+        detector = cms.string("HCAL_ENDCAP"),
+        depths = cms.vint32(1, 2, 3, 4, 5, 6, 7),
+        seedingThreshold = cms.vdouble(0.1375, 0.275, 0.275, 0.275, 0.275, 0.275, 0.275),
```

Comment:
please point me to the slides that justify this selection.

Reply:
I kept the same same ratio seed/gathering threshold for each recHit. jetMET agreed
Based on noise we only decided on the recHit/gatheringthreshold

Reply:
OK, makes sense.